Neos.Neos/Documentation/References/NodeMigrations.rst

```diff
@@ -199,11 +196,27 @@ Options Reference:
   The value of this option (defaults to ``{current}``) will be used to include the current property value into the new
   value.
 
+MoveDimensionSpacePoint
+~~~~~~~~~~
+
+Moves a dimension space point globally.
+
+``from`` (array)
+  Source Dimension Space Point as array. E.g. ["language" => "es", "country" => "es"]
+``to`` (array)
+  Target Dimension Space Point as array. E.g. ["language" => "es", "country" => "ar"]
+
+
 RemoveNode
```

Comment:
Why isn't this one called `RemoveNodeAggregate`? `RenameNode` is now called `RenameNodeAggregate`.

Reply:
IDK! We can discuss that, but this just documents the current state.

Reply:
might need another issue / task then :)